rq/worker.py

```diff
@@ -734,8 +734,12 @@ class HerokuWorker(Worker):
     causes the horse to crash `imminent_shutdown_delay` seconds later
     """
     imminent_shutdown_delay = 6
-    frame_properties = ['f_code', 'f_exc_traceback', 'f_exc_type', 'f_exc_value',
-                        'f_lasti', 'f_lineno', 'f_locals', 'f_restricted', 'f_trace']
+
+    frame_properties = ['f_code', 'f_lasti', 'f_lineno', 'f_locals', 'f_trace']
+    if sys.version_info[:2] < (3, 0):
```

Comment:
Would have been good to get a chance to review this, surely no need to merge it immediately?

Would it not have been better to use `if PY2` from `compat`? That's surely what `compat` is there for.